mlrun/api/api/utils.py

```diff
@@ -679,6 +679,13 @@ def _submit_run(
         # fn.spec.rundb = "http://mlrun-api:8080"
         schedule = data.get("schedule")
         if schedule:
+            if data.get("overwrite_schedule"):
+                get_scheduler().delete_schedule(
+                    db_session,
+                    task["metadata"]["project"],
+                    task["metadata"]["name"],
+                )
+                db_session.commit()
             cron_trigger = schedule
             if isinstance(cron_trigger, dict):
                 cron_trigger = schemas.ScheduleCronTrigger(**cron_trigger)
```

Comment:
In line 726 there is `logger.info("Run submission succeeded", response=response)` which will happen even if the run failed. I think this print shouldn't happen in case of exception.

Reply:
but if there is an exception `log_and_raise` raises the error